api/overview/azure/batch.md

```diff
@@ -47,10 +87,40 @@ Install-Package Microsoft.Azure.Management.Batch.Fluent
 dotnet add package Microsoft.Azure.Management.Batch.Fluent
 ```
 
+### Example
+
+The following example retrieves the quota for the subscription, creates an account, and regenerates the primary account key.
+
+```csharp
+/*
+using Microsoft.Azure.Management.Batch;
+using Microsoft.Azure.Management.Batch.Models;

```

Comment:
Need to update to the fluent namespaces.

using Microsoft.Azure.Management.Batch.Fluent;
using Microsoft.Azure.Management.Batch.Fluent.Models;
using Microsoft.Rest;